tests/gdmem/malloc.c

```diff
@@ -0,0 +1,77 @@
+#include "gd.h"
+#include "gdtest.h"
+
+void *testcalloc(size_t nmemb, size_t size)
+{
+	char *ptr;
+
+	ptr = calloc(nmemb, size);
+
+	for (size_t i = 0; i < size; i++)
+		*(ptr + i) = 0;
```

Comment:
`calloc()` is documented as doing this, so why do we need to ?

Reply:
It was done for my tests, but I'd rather go to a much proper check.